src/controllers/softtakeover.cpp

```diff
@@ -60,14 +66,21 @@ bool SoftTakeoverCtrl::ignore(ControlObject* control, double newParameter) {
     return ignore;
 }
 
+void SoftTakeoverCtrl::ignoreNext(ControlObject* control) {
```

Comment:
not with pointers. The style guide isn't explicit here, but I find == NULL more clear and less prone to error